index.html

```diff
@@ -0,0 +1,21 @@
+---
+dateModified: "2025-02-12"
+language: "en"
+title: "OAG - template"
+layout: core
+pageclass: "oag-theme"
+noFooterMain: true
+breadcrumbs: false
+---
+
+<div class="container">	
+	<div class="row wb-eqht-grd">	
+		<h2 class="col-xs-12 h4  text-center mb-4 mt-5">Office of the Auditor General of Canada template</h2>
```

Comment:
Separate the layout from the semantic. That heading should not be part of the grid column. Division should be used instead and the heading could be inside.